content/en/admin/config.md

```diff
@@ -546,6 +546,11 @@ You must serve the files with CORS headers, otherwise some functions of Mastodon
 
 #### `OMNIAUTH_ONLY`
 
+#### `ONE_CLICK_SSO_LOGIN`
+Enables the `Login or Register` button.
+Useful for instances where all authentication takes place using a single
+external provider (CAS, SAML or OIDC).
+
```

Comment:
I'd mention that it may prevent the server from starting up if the OIDC server is not available, and also that it prevents caching of anonymous HTML pages.

Reply:
Done. I added that the start up would only fail when using OIDC discovery, as the other authentication methods are unaffected.